cmake/Hunter/config.cmake

```diff
@@ -39,10 +39,32 @@ hunter_config(
 )
 
 if ("${WASM_COMPILER}" STREQUAL "WasmEdge")
+  hunter_config(
+      fmt
+      URL
+          https://github.com/fmtlib/fmt/archive/refs/tags/10.2.1.tar.gz
+      SHA1
+          d223964b782d2562d6722ffe67027204c6035453
+      CMAKE_ARGS
+          CMAKE_POSITION_INDEPENDENT_CODE=TRUE
+  )
+
+  hunter_config(
+      spdlog
+      VERSION 1.12.0-p0
+      CMAKE_ARGS
+          SPDLOG_BUILD_PIC=ON
+          SPDLOG_FMT_EXTERNAL=ON
+  )
+
+  # hash or version, required later to separate WasmEdge 
+  # binary cache directories from different versions
+  set(WASMEDGE_ID 58aea400de9179ad3e314c7e84fd4da345b8a643)
```

Comment:
I don't get the idea behind dedicating the id to a separate variable.

Reply:
it is needed to use this ID in C++ code to add it to the path of the runtime cache. That way when wasmedge version will be updated, a new path to runtime caches will be generated to avoid possible conflicts between incompatible runtimes

Reply:
And a variable is needed simply because we use this ID in two places in CMakeLists, and if it's not in a variable it will be forgotten in one of them when we change it.